.packit.yaml

```diff
@@ -101,3 +101,14 @@ jobs:
     tmt_plan: plans/onboarding
     packages: [fido-device-onboard-centos]
     targets: ["centos-stream-9", "centos-stream-10"]
+
+  # RHEL-10 jobs
+
+  - job: tests
+    trigger: pull_request
+    identifier: onboarding-image-mode
+    fmf_path: test/fmf
+    tmt_plan: plans/image-mode-onboarding
+    use_internal_tf: true
+    packages: [fido-device-onboard-centos]
```

Comment:
Maybe I am wrong but if you are going to build the package in the spawned VM you don't need to specify `packages`, just use `skip_build: true` to run the tests without a copr build.

Reply:
I'm skipping package building in the spawned VM, to make use of the copr build. But the issue I'm having is that rhel-10 job is not being triggered.